clusters/app.ci/pools/openshift-ci/ci-ocp-4-8-0-fc.4-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: ci-ocp-4-8-fc4-amd64-aws-us-east-1
```

Comment:
Are you going to create a new pool when a new version is out?
IMHO, the pool name reflects Y stream.
We could ref another imageSet for the new version in 4.8.